File: 0412-fizz-buzz/0412-fizz-buzz.py
class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        res=[]
        for i in range(1,n+1):
            st=''
            if i %3==0:
                st +='Fizz'
            if i%5==0:
                st +='Buzz'
            if st=='':
                st=str(i)
            res.append(st)
        return res
        

File: 0412-fizz-buzz/0412-fizz-buzz.js
/**
 * @param {number} n
 * @return {string[]}
 */
var fizzBuzz = function(n) {
    let arr=[]
    for (let i=1;i<=n;i++){
        let str=''
        if (i%3===0){
            str +='Fizz'
        }
        if(i%5===0){
            str +='Buzz'
        }
        if (str===''){
            arr.push(`${i}`)
        }else{
            arr.push(str)
        }
        
    }
    return arr
};